unfetter-discover/unfetter
# new deployed group in hosts.ini controls what is deployed

Whatever hosts are under the new [deployed] group in the hosts.ini file will be used for deployment.

TO TEST:
Add one or more hosts (dev, dev-demo, prod-uac, etc) under the [deployed] group
run "ansible-playbook deploy.yml --list-hosts"   
You should see a list of whatever hosts you put under [deployed]

run "ansible-playbook deploy-dev.yml --list-hosts" should still show only the "dev" host
